tests/e2e/support/objects/app-admin-settings/users/actions.ts

```diff
@@ -253,14 +253,24 @@ export const changeUser = async (args: {
   uuid: string
   attribute: string
   value: string
-}): Promise<void> => {
+}): Promise<unknown> => {
   const { page, attribute, value, uuid } = args
   await page.locator(util.format(userInput, attribute)).fill(value)
 
   if (attribute === 'role') {
     await page.locator(util.format(roleValueDropDown, value)).click()
+    return Promise.all([
+      page.waitForResponse(
```

Comment:
not sure if promise.all could make problems in case one requests fails

Reply:
this is intended and necessary

Reply:
ok then its fine 👍🏼